content/blog/authors/krysal/contents.lr

```diff
@@ -0,0 +1,8 @@
+username: krysal
+---
+name: Krystle Salazar
+---
+md5_hashed_email:
```

Comment:
If you feel comfortable with it, please add your md5 hashed email address here. You can generate your hashed email using this md5 hash generator: https://www.md5hashgenerator.com/ :)

Reply:
Done! :)